lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.ts

```diff
@@ -321,14 +317,9 @@ export class ProcessListCloudComponent extends DataTableSchema implements OnChan
     }
 
     onColumnOrderChanged(columnsWithNewOrder: DataColumn[]): void {
-        this.columnsOrder = columnsWithNewOrder.map(column => column.id);
-
         if (this.appName) {
-            this.cloudPreferenceService.updatePreference(
-                this.appName,
-                ProcessListCloudPreferences.columnOrder,
-                this.columnsOrder
-            );
+            const newColumnsOrder = columnsWithNewOrder.map(column => column.id);
+            this.processListCloudService.updateColumnOrderPreferences(this.appName, newColumnsOrder);
```

Comment:
Why moving this into the service where we already have this available here? isn't an overcomplication?